std/internal/math/gammafunction.d

```diff
@@ -29,7 +29,7 @@ nothrow:
 
 private {
 
-enum real SQRT2PI = 2.50662827463100050242E0L; // sqrt(2pi)
+enum real SQRT2PI = 2.5066282746310005024157652848110452798109E0L; // sqrt(2pi)
```

Comment:
Oh nice :)
http://www.wolframalpha.com/input/?i=sqrt(2pi)+in+base+16
Cut wherever you want:

```
0x2.81b263fec4e0b2caf9483f5ce459dc5f19f3ea6416000b50dc2f412ddeeb948b5068337b65698726847d560b8818fb98307f0f37d42a17c1c2b86baec9ffa
```